.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+repos:
```

Needs a global exclude for the license file. These are legal texts and should not be reformatted.

(Untested)
```suggestion
exclude: ^LICENSE
repos:
```